Jellyfin.Data/Enums/ViewType.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace Jellyfin.Data.Enums
+{
+    /// <summary>
+    /// An enum representing the type of view for a library or collection.
+    /// </summary>
+    public enum ViewType
+    {
+        /// <summary>
+        /// Shows banners.
+        /// </summary>
+        Banner = 0,
+
+        /// <summary>
+        /// Shows a list of content.
+        /// </summary>
+        List = 1,
+
+        /// <summary>
+        /// Shows poster artwork.
+        /// </summary>
+        Poster = 2,
+
+        /// <summary>
+        /// Shows poster artwork with a card containing the name and year.
+        /// </summary>
+        PosterCard = 3,
+
+        /// <summary>
+        /// Shows a thumbnail.
+        /// </summary>
+        Thumb = 4,
```

Comment:
Can we rename this to `Thumbnail`?

Reply:
I named it this because that is what web sends us, we could rename it but we would need additional parsing code or changes to web.